src/frontend/src/views/Faq/index.tsx

```diff
@@ -215,12 +228,20 @@ export default function Faq(): JSX.Element {
         For new samples that you do not choose to mark &quot;private,&quot; we
         share this data in two ways:
       </P>
-      <List
-        items={[
-          `After two weeks, we will automatically submit the sample to GISAID on your behalf. This upload includes the pathogen genome; collection date; sequencing lab (i.e., your group); and location (as always, at the county level or above). `,
-          `Consistent with the prior COVID Tracker program’s policies, the California Department of Public Health (CDPH) will be able to see non-private samples, but with your internal, private identifiers redacted.`,
-        ]}
-      />
+      <List>
+        <ListItem>
+          After two weeks, we will automatically submit the sample to GISAID on
```

Comment:
Not a code thing, but I wonder if we should still have this? We don't actually do this right now, right?

Reply:
(Not as part of this PR, but maybe just like a note we should take down and figure out if a change is needed next standup?)

Reply:
i think we should just remove it for now.